pkg/opensearch/time_series_query_test.go

```diff
@@ -209,6 +209,58 @@ func TestExecuteTimeSeriesQuery(t *testing.T) {
 			assert.Equal(t, "asc", termsAgg.Order["_key"])
 		})
 
+		t.Run("With term agg and order by term with OpenSearch, _term is replaced by _key", func(t *testing.T) {
+			c := newFakeClient(es.OpenSearch, "1.0.0")
+			_, err := executeTsdbQuery(c, `{
+				"timeField": "@timestamp",
+				"bucketAggs": [
+					{
+						"type": "terms",
+						"field": "@host",
+						"id": "2",
+						"settings": { "size": "5", "order": "asc", "orderBy": "_term"	}
+					},
+					{ "type": "date_histogram", "field": "@timestamp", "id": "3" }
+				],
+				"metrics": [
+					{"type": "count", "id": "1" },
+					{"type": "avg", "field": "@value", "id": "5" }
+				]
+			}`, from, to, 15*time.Second)
+			assert.NoError(t, err)
+			sr := c.multisearchRequests[0].Requests[0]
+
+			firstLevel := sr.Aggs[0]
+			assert.Equal(t, "2", firstLevel.Key)
+			termsAgg := firstLevel.Aggregation.Aggregation.(*es.TermsAggregation)
+			assert.Equal(t, "asc", termsAgg.Order["_key"])
+		})
+
+		t.Run("With term agg and valid min_doc_count", func(t *testing.T) {
```

Comment:
Argh sorry I meant to put this test with this PR https://github.com/grafana/opensearch-datasource/pull/268 (it has already been merged)